app/src/main/java/org/mifos/mobile/ui/activities/HomeActivity.kt

```diff
@@ -148,7 +149,7 @@ class HomeActivity : BaseActivity(), UserDetailsView, NavigationView.OnNavigatio
             R.id.item_recent_transactions -> replaceFragment(RecentTransactionsFragment.newInstance(), true, R.id.container)
             R.id.item_charges -> replaceFragment(ClientChargeFragment.newInstance(clientId, ChargeType.CLIENT), true,
                     R.id.container)
-            R.id.item_third_party_transfer -> replaceFragment(ThirdPartyTransferFragment.newInstance(), true, R.id.container)
+            R.id.item_third_party_transfer -> startActivity(Intent(this@HomeActivity, TransactionActivity::class.java))//replaceFragment(ThirdPartyTransferFragment.newInstance(), true, R.id.container)
```

Comment:
- what about `ThirdPartyTransferFragment`, is it needed now, or can be removed?
- why don't we use Fragment here, like done for others?

Reply:
I am also not much clear about point 1 (Where to replace the PaymentHub Activity). 
@garvit984 Can you please clear this.

Reply:
Now, there is no such thing as Third-party transfer explicitly. For different tenants and parties, all the logic is handled by the Payment Hub Connector now. So for the front-end application, we don't have to do much.

Reply:
- why don't we use Fragment here, like done for others? clarify @Sparsh1212

Reply:
1. We have a presenter and contract for it, therefore it has to be made in an `Activity`.
2. Also, its not related to the other fragments and is a completely separate thing in itself.
So, because of these two reasons, we have to make this an activity instead of the fragment unlike other cases.
I hope this clarifies.